runtime/runtime.go

```diff
@@ -174,14 +142,27 @@ func (r *interpreterRuntime) ExecuteScript(script Script, context Context) (cade
 
 	runtimeStorage := newRuntimeStorage(context.Interface)
 
-	functions := r.standardLibraryFunctions(context, runtimeStorage)
+	var checkerOptions []sema.Option
+	var interpreterOptions []interpreter.Option
```

Comment:
Are these meant to be empty here?

Reply:
Yes, I declared those and gave them descriptive names, instead of just passing `nil` literals as arguments, to make it explicit what the arguments are / mean.